src/prototype_room_creepbuilder.js

```diff
@@ -285,7 +288,7 @@ Room.prototype.getPartConfig = function(creep) {
   if (!sufix.fail && !sufix.null) {
     parts = parts.concat(sufix.parts);
   }
-  if (config.debug.spawn) {
+  if (config.debug.spawn && !countOnly) {
```

Comment:
Before introducing a new parameter, I would rephrase the log message, like replacing `Spawning` with `getPartConfig`.

It is just a debug statement.